standards.rst

```diff
@@ -0,0 +1,15 @@
+.. _standards:
+
+Standardized naming conventions
+===============================
+
+The IAMC community and the *Scenario Services & Scientific Software team* in the Energy,
```

Comment:
For more advertising, you could put a link to the _Scenario Services & Scientific Software team_ on the IIASA website

Reply:
Nice suggestion, added a shorthand |ScSe team| to be used consistently across the pages.